`yarn build` で dune が必要だったので、それらも ocaml-general-gha の中で呼んでいます。

https://github.com/kxcdev/kxc-melange-react-native-template/actions/runs/6954173267/job/18920513175

```
$ turbo build
• Packages in scope: MelangeRNTemplate, mobile-app-ui, tsconfig
• Running build in 3 packages
• Remote caching disabled
mobile-app-ui:build
MelangeRNTemplate:build
  cache miss, executing 24e66b9ea211ba1e
  $ dune build .
  Error: MelangeRNTemplate#build: command (/home/runner/work/kxc-melange-react-native-template/kxc-melange-react-native-template/jsland/apps/mobile-app) yarn run build exited (127)
   ERROR  run failed: command  exited (127)
  /bin/sh: 1: dune: not found
  error Command failed with exit code 127.
  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
  Error:  command finished with error: command (/home/runner/work/kxc-melange-react-native-template/kxc-melange-react-native-template/jsland/apps/mobile-app) yarn run build exited (127)

 Tasks:    1 successful, 2 total
Cached:    0 cached, 2 total
  Time:    2.407s 
Failed:    MelangeRNTemplate#build

error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 127.
```

--- a/.github/workflows/common.yml
+++ b/.github/workflows/common.yml
@@ -14,6 +14,7 @@ jobs:
           - 5.1.0
         node-version:
           - hydrogen
+    if: github.repository == 'kxcdev/kxc-melange-react-native-template'
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout code

--- a/.github/workflows/common.yml
+++ b/.github/workflows/common.yml
@@ -34,6 +34,7 @@ jobs:
           ocaml-version: ${{ matrix.ocaml-version }}
           node-version: ${{ matrix.node-version }}
           setup-command: "opam install --locked . -y --deps-only --with-test"
+          # Since `dune` was needed to run `yarn build`, they are also called here.
           build-command:
             "(dune build && cd jsland && yarn install --frozen-lockfile && yarn build)"
           pr-report-test-result: ${{ matrix.ocaml-version == '5.1.0' }}

--- a/.github/workflows/common.yml
+++ b/.github/workflows/common.yml
@@ -33,8 +33,8 @@ jobs:
         with:
           ocaml-version: ${{ matrix.ocaml-version }}
           node-version: ${{ matrix.node-version }}
-          setup-command: "opam install --locked . -y --deps-only --with-test"
+          setup-command: "opam install --locked . -y --deps-only --with-test && (cd jsland && yarn install --frozen-lockfile)"
           # Since `dune` was needed to run `yarn build`, they are also called here.
           build-command:
-            "(dune build && cd jsland && yarn install --frozen-lockfile && yarn build)"
-          pr-report-test-result: ${{ matrix.ocaml-version == '5.1.0' }}
+            "(dune build && cd jsland && yarn build)"
+          pr-report-test-result: ${{ matrix.ocaml-version == '5.1.0' && matrix.node-version == 'hydrogen' }}

--- a/README.md
+++ b/README.md
@@ -166,6 +166,18 @@ dune build
 ![Android Screen Shot - Intro Screen](./docs/images/sc-android-1.png "Intro Screen")
 ![Android Screen Shot - Accumulator Example](./docs/images/sc-android-2.png "Accumulator Example")
 
+## Feature
+
+### Github Actions
+
+Github Actions can be enabled by deleting the following statement in ``.github/workflows/common.yml``.
+
+```yml
+if: github.repository == 'kxcdev/kxc-melange-react-native-template'
+```
+
+This is an opt-in feature because it uses [kxcdev/ocaml-general-gha](https://github.com/kxcdev/ocaml-general-gha) instead of the official [setup-ocaml](https://github.com/ocaml/setup-ocaml).
+
 ## License
 Most code is licensed under Apache License, Version 2.0 (the "License")
 by the copyright holder(s) listed below.

